purchase_report_qweb_decimal_place/reports/purchase_report.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=deprecated-data-xml-node -->
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//span[@t-field='line.price_unit']" position="replace">
+            <t t-set="currency" t-value="o.currency_id" />
+            <t t-set="price_unit" t-value="line.price_unit" />
+            <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+        </xpath>
```

Comment:
Depends on this discussion: https://github.com/qrtl/fal-custom/pull/6#pullrequestreview-1236823885